camel/toolkits/openbb_toolkit.py

```diff
@@ -0,0 +1,1126 @@
+            beta_max (Optional[float]): Max beta value
+            provider (str): Data provider
+            return_type (Literal["df", "raw"]): Return format
+
+        Returns:
+            Union[pd.DataFrame, Dict]: Screener results
+        """
+        try:
+            params = {
+                k: v
+                for k, v in {
+                    'country': country,
+                    'exchange': exchange,
+                    'sector': sector,
+                    'industry': industry,
+                    'market_cap_min': market_cap_min,
+                    'market_cap_max': market_cap_max,
+                    'beta_min': beta_min,
+                    'beta_max': beta_max,
+                }.items()
+                if v is not None
+            }
+
+            data = self.client.equity.screener.screen(
+                provider=provider, **params
+            )
+
+            if return_type == "df":
+                return data.to_df()
+            return data.results
+        except Exception:
+            if return_type == "df":
+                return pd.DataFrame(
+                    columns=[
+                        'symbol',
+                        'name',
+                        'market_cap',
+                        'sector',
+                        'industry',
+                    ]
+                )
+            return {}
+
+    def get_available_indices(
+        self,
+        provider: str = "yfinance",
+        return_type: Literal["df", "raw"] = "df",
+    ) -> Union[pd.DataFrame, Dict]:
+        """Get list of available market indices.
+
+        Args:
+            provider (str): Data provider
+            return_type (Literal["df", "raw"]): Return format
+
+        Returns:
+            Union[pd.DataFrame, Dict]: Available indices with their symbols
+        """
+        data = self.client.index.available(provider=provider)
+        return data.to_df() if return_type == "df" else data.results
+
+    def get_stock_quote(
+        self,
+        symbol: str,
+        source: str = "iex",
+        return_type: Literal["df", "raw"] = "df",
+    ) -> Union[pd.DataFrame, Dict]:
+        """Get current stock quote for a given symbol.
+
+        Args:
+            symbol: Stock symbol (e.g., 'AAPL' for Apple Inc.)
+            source: Data source ('iex', 'bats', 'bats_delayed', 'utp_delayed',
+                   'cta_a_delayed', 'cta_b_delayed', 'intrinio_mx',
+                   'intrinio_mx_plus', 'delayed_sip')
+            return_type: Return format ('df' or 'raw')
+
+        Returns:
+            Stock quote data in requested format
+        """
+        try:
+            data = self.client.equity.price.quote(symbol=symbol, source=source)
+            return data.to_df() if return_type == "df" else data.results
+        except Exception as e:
+            msg = f"Failed to get stock quote for {symbol}: {e}"
+            print(msg)
+            return pd.DataFrame() if return_type == "df" else {}
```

Comment:
use `logger` here instead of `print`， **same as other parts**

Reply:
Thanks @Wendong-Fan , just replaced all `print` with `logger`